<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>416</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBoxSplit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>471</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Разделить изображение (512 на 512)</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>19</y>
      <width>451</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="groupBoxSplitLayout">
     <item>
      <widget class="QPushButton" name="getImageForSplitBtn">
       <property name="text">
        <string>Разделить изображение</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxRun">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>471</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>Настройки запуска</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>451</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="groupBoxRunLayout">
     <item>
      <widget class="QPushButton" name="getResultBtn">
       <property name="text">
        <string>Выполнить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopProcessBtn">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>451</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="groupBoxSavePathLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="horizontalSpacing">
      <number>7</number>
     </property>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditSavePath"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="setPathToVectorBtn">
       <property name="maximumSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelSavePath">
       <property name="text">
        <string>Путь сохранения:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>451</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="groupBoxSelectImagesLayout">
     <item row="0" column="2">
      <widget class="QPushButton" name="getPathsImagesBtn">
       <property name="text">
        <string>Выбрать изображения</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelCountSelectedImages">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Количество выбранных изображений:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditSelectedImages"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>451</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="groupBoxPathToModelLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="horizontalSpacing">
      <number>7</number>
     </property>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditPathToModel"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="setPathToModelBtn">
       <property name="maximumSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelPathToModel">
       <property name="text">
        <string>Путь до модели:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="labelRemainingTime">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>461</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>До завершения осталось: 00 : 00 : 00</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
